editor/quick_open.cpp

```diff
@@ -29,24 +29,20 @@
 /*************************************************************************/
 
 #include "quick_open.h"
-
```

Comment:
This should not be removed. https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes